iml-agent/src/action_plugins/check_stonith.rs

```diff
@@ -376,4 +392,20 @@ mod tests {
             }
         );
     }
+
+    #[test]
+    fn test_stonith_exascalar_static() {
+        let testxml = include_bytes!("snapshots/check_stonith_test_es.xml");
```

Comment:
is snapshots the right place for this? We probably shouldn't write anything there since it's managed by `insta`.

How about `fixtures` instead?

Reply:
okay.  I think I'll need to change one of the other patches then.